python/idsse_common/idsse/common/rabbitmq_utils.py

```diff
@@ -457,7 +482,7 @@ def __init__(
                                 exclusive=True,
                                 auto_delete=False,
                                 arguments={'x-queue-type': 'classic',
-                                           'x-message-ttl': 10 * 1000})
+                                            'x-message-ttl': 10 * 1000})
```

Comment:
```suggestion
                                           'x-message-ttl': 10 * 1000})
```
I don't think this should be indented an extra space, is it?

Reply:
Hm, I'm not sure if that was even intentional.